x-pack/plugins/integration_assistant/server/graphs/cel/graph.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { StateGraphArgs } from '@langchain/langgraph';
+import { END, START, StateGraph } from '@langchain/langgraph';
+import type { CelInputState } from '../../types';
+import { handleBuildProgram } from './build_program';
+import { handleGetStateDetails } from './retrieve_state_details';
+import { handleGetStateVariables } from './retrieve_state_vars';
+import { handleSummarizeQuery } from './summarize_query';
+import { CelInputBaseNodeParams, CelInputGraphParams } from './types';
+
+const graphState: StateGraphArgs<CelInputState>['channels'] = {
+  lastExecutedChain: {
+    value: (x: string, y?: string) => y ?? x,
+    default: () => '',
+  },
+  dataStreamName: {
+    value: (x: string, y?: string) => y ?? x,
+    default: () => '',
+  },
+  finalized: {
+    value: (x: boolean, y?: boolean) => y ?? x,
+    default: () => false,
+  },
+  results: {
+    value: (x: object, y?: object) => y ?? x,
+    default: () => ({}),
+  },
+  apiDefinition: {
+    value: (x: string, y?: string) => y ?? x,
+    default: () => '',
+  },
+  apiQuerySummary: {
+    value: (x: string, y?: string) => y ?? x,
+    default: () => '',
+  },
+  exampleCelPrograms: null,
```

Comment:
Why `null` ?

```suggestion
  exampleCelPrograms: {
    value: (x: string[], y?: string[]) => y ?? x,
    default: () => [],
  },
```

Reply:
I'm not sure, I guess I missed filling that one in.